Roles/Crewmate/Crusader.cs

```diff
@@ -26,11 +26,12 @@ public static void Init()
     {
         playerIdList = new();
         CrusaderLimit = new();
+        CurrentKillCooldown = new();

```

Comment:
CurrentKillCooldown did not reset after a new game, and accordingly this led to a critical error, my friends after these fixes no longer received any errors